src/main/resources/static/api.yaml

```diff
@@ -1561,6 +1561,103 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/forgot-pin/generate-otp:
+    post:
+      tags:
+        - account
+      summary: Generate OTP for forgot pin flow
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+      responses:
+        '201':
+          description: OTP generated successfully.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OTPSessionIdResponse'
+        '500':
+          description: >
+            **Causes:**
+              * Downstream services are down
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /patients/forgot-pin/validate-otp:
+    post:
+      tags:
+        - account
+      summary: Validate OTP for forgot pin flow
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VerifyOtpRequest'
+      responses:
+        '201':
+          description: OTP validation successful.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VerifyOtpResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/VerifyOtpResponse'
+        '500':
+          description: >
+            **Causes:**
+              * Downstream services are down
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /patients/forgot-pin/update-pin:
```

Comment:
there are only two things right, one I know my existing pin and try to update the pin, or reset-pin.

URL could be as simple as /patients/reset-pin

Reply:
Yes, I can be. We will change it.